### PR TITLE
Fixing the icons in the RoleSelectOptions

### PR DESCRIPTION
**Pull request by Tanosimboangy**
Hello Team,

Team Fannar has just finished fixing the Role Select Option. Please have a look at it [here](https://github.com/onja-org/vamalaka-fannar/compare/FixingRoleSelectOption?expand=1) Any comment would be highly appreciated. 

Thanks in advance.

### CODE REVIEW

**Review by mitia-anah — APPROVED** [review]

File: src/components/RoleIcon/Seller.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
```
Comment (andrzej-onja) [inline src/components/RoleIcon/Seller.tsx:1]: same as buyer just move to another more generic folder Icons

File: src/components/RoleSelectOption/RoleSelectOption.stories.tsx [hunk 2]
```diff
@@ -1,30 +1,51 @@
...
+//     alt:"User icon",
+// };
 
+// export const Seller = Template.bind({});
+// Seller.args = {
+//     label:"Seller",
+//     text:"Own or belong to a company, this is for you.",
+//     src: briefcaseIcon,
+//     alt:"Briefcase icon",
+// };
```
Comment (andrzej-onja) [inline src/components/RoleSelectOption/RoleSelectOption.stories.tsx:51]: remove commented out code

File: src/components/RoleSelectOption/RoleSelectOption.tsx [hunk 3]
```diff
@@ -1,129 +1,249 @@
...
+
+//   @media (max-width: 360px) {
+//     text-align: center;
+//     margin: 0;
+//   }
+
+//   @media (max-width: 476px) {
+//     max-width: 150px;
+//   }
+// `
```
Comment (andrzej-onja) [inline src/components/RoleSelectOption/RoleSelectOption.tsx:249]: remove commented out code

File: .vscode/settings.json [hunk 4]
```diff
@@ -1,2 +1,3 @@
 {
-}
\ No newline at end of file
+  "liveServer.settings.port": 5501
+}
```
Comment (ganamavo) [inline .vscode/settings.json:3]: I don't know if this is worth being pushed or just ignored

File: src/components/LeftSide/LeftSide.tsx [hunk 5]
```diff
@@ -1,42 +1,42 @@
 import styled from 'styled-components'
-import { fonts } from '../../globalStyles/fonts';
-import Baobab from '../LeftSide/images/baobab.png';
+import { fonts } from '../../globalStyles/fonts'
+import Baobab from '../LeftSide/images/baobab.png'
```
Comment (ganamavo) [inline src/components/LeftSide/LeftSide.tsx:3]: looks like you're importing a component but actually an image. Maybe should make all lowercase. I don't know about the project's standard though but it's not a big deal

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**